hail/python/deploy.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -ex
+
+cd $(CDPATH= cd -- "$(dirname -- "$0")" && pwd)
+python3=${HAIL_PYTHON3:-python3}
+
+published=$(
+    pip --no-cache-dir search hail \
+     | grep '^hail ' \
+     | sed 's/hail (//' \
+     | sed 's/).*//')
+current=$(
```

Comment:
I don't love this. We have the version defined in build.gradle - let's just pass it as an argument into this script, or set it as an environment flag.

Reply:
Hmm. I see what you mean about this, but I'm also not a fan of having two sources of truth. If something gets out of sync we could deploy a version of hail whose `_generated_version_info.py` disagrees with PyPI's version.

Reply:
I could pass it in and verify it matches this expression exactly?